bodhi/server/models.py

```diff
@@ -1339,6 +1433,21 @@ class ModuleBuild(Build):
         'polymorphic_identity': ContentType.module,
     }
 
+    @property
```

Comment:
Can't you use `['name']`, `['version']` and `['release']` also in these properties in the `Build` class and therefore you could remove these properties completely from `ModuleBuild` class and also remove `_get_nvr` from the `Build` class?

Reply:
No, because the way you get nvr_name etc are entirely different for modules versus everything else.